openkfw/TruBudget
# #1755 api: fetch user groups from server

### Checklist




### Description


### How to test

To test if these changes prevent the fault of disabling authentication, you need to reproduce the bug:

0. On main branch:
1. Create lots of groups with long names, so so the token cookie will exceed the limit of ~4kBs, with user XYZ assigned to each (example below, add to groups.json and rebuild provisioning image)
2. Login as XYZ
3.  You should not be able to login (browser ignores set-sookie header if it's over 5000 characters)
4. Switch to MR branch, same setup. You should be able to login




Closes #1755
